lte/gateway/docker/docker-compose.yml

```diff
@@ -0,0 +1,232 @@
+---
+version: "3.7"
+
+# Standard logging for each service
+x-logging: &logging_anchor
+  driver: ${LOG_DRIVER}
+
+# Standard volumes mounted
+x-standard-volumes: &volumes_anchor
+  - ${CONFIGS_DEFAULT_VOLUME}:/etc/magma/
```

Comment:
Should these configuration files be in the "configs" sub-section instead of volumes?

Reply:
these are default config copied from repo.